docs/unattend.xml

```diff
@@ -16,18 +16,14 @@
          <CEIPEnabled>0</CEIPEnabled>
       </component>
       <component xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="Microsoft-Windows-Shell-Setup" processorArchitecture="amd64" publicKeyToken="31bf3856ad364e35" language="neutral" versionScope="nonSxS">
-         <ComputerName>windows-host</ComputerName>
```

Comment:
@codyhoag please review this commit. We need this reflected in [Creating the vSphere Windows VM golden image](https://docs.openshift.com/container-platform/4.8/windows_containers/creating_windows_machinesets/creating-windows-machineset-vsphere.html#creating-the-vsphere-windows-vm-golden-image_creating-windows-machineset-vsphere)

Reply:
Done